src/main/java/io/strimzi/kafka/bridge/mqtt/mapper/MqttKafkaMapper.java

```diff
@@ -14,14 +14,15 @@
 public abstract class MqttKafkaMapper {
 
     // default Kafka topic. Used when no mapping rule matches the mqtt topic.
-    public static final String DEFAULT_KAFKA_TOPIC = "messages_default";
+    public static final String MAPPER_DEFAULT_KAFKA_TOPIC = "messages_default";
```

Comment:
you are passing the default topic to the mapper so the default should not be here anymore.
It should live in the  `BridgeConfig` class so that it doesn't need a dependency on the mapper.
That is the only place where the default topic constant is used.

Reply:
I agree. It is doable!